syntax = "proto3";

import "trade/vo/futures/FuturesFundInfoVo.proto";
option
java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response.futures";
option java_outer_classname = "FuturesQueryFundInfoResponseProto";
message FuturesQueryFundInfoResponse {
	FuturesFundInfoVo fundInfo = 1;
}
